content/go/go.md

```diff
@@ -9,4 +9,4 @@ CatalogContent:
 
 Go (also referred to as "Golang") is an open-source compiled programming language that was designed at Google by Robert Griesemer, Rob Pike, and Ken Thompson. It had its initial release for the public in 2012.
```

Comment:
```suggestion
**Go** (also referred to as "Golang") is an open-source compiled programming language that was designed at Google by Robert Griesemer, Rob Pike, and [Ken Thompson](https://www.codecademy.com/resources/docs/general/ken-thompson). It had its initial release for the public in 2012.
```

Reply:
For the `Description` in line # 3, I'd also suggest shortening to one sentence like this:

'Go (or Golang) is an open-source compiled programming language designed at Google and released to the public in 2012.'